src/snowflake/cli/plugins/object/commands.py

```diff
@@ -21,6 +21,17 @@
 ObjectArgument = typer.Argument(
     help="Type of object. For example table, procedure, streamlit.",
     case_sensitive=False,
+    show_default=False,
+)
+ObjectDefinitionArgument = typer.Argument(
+    help="""Object definition in JSON format, for example
+\'{"name": "my_db", "comment": "created with Snowflake CLI"}\', or provided as a list of key=value pairs,
+for example: name=my_db 'comment=created with Snowflake CLI'.
+
+Check https://docs.snowflake.com/LIMITEDACCESS/rest-api/reference/ for the full list of available parameters
```

Comment:
Do not link to private docs, we should not merge the changes as long as APIs are not in PuPr.

Reply:
I left TODO to be filled, and changed base branch from main to `object-create`, which will collect PRs and merge them together into main when APIs will gave gone PuPr.